capa/features/address.py

```diff
@@ -56,48 +54,48 @@ def __repr__(self):
 class DNTokenAddress(Address):
     """a .NET token"""
 
-    def __init__(self, token: Token):
+    def __init__(self, token: int):
         self.token = token
```

Comment:
I think we can update these like the other classes, adding:
`return int.__new__(cls, token)`

Reply:
c5b4fc33

Reply:
c5b4fc33

Reply:
updated